requirements.txt

```diff
@@ -7,7 +7,8 @@ Pillow
 pycairo
 pyenchant
 pygobject
-# pygtkspelaplcheck
+# pygtkspellcheck
 reportlab
+scrape-schema-recipe
 selenium
-sqlalchemy
\ No newline at end of file
+sqlalchemy
```

Comment:
Can you tell us more about these changes? Why is sqlalchemy now a dependency?

Reply:
`sqlalchemy` has already been a dependency before. I just fixed the naming for `pygtkspellcheck` and added the `scrape-schema-recipe` requirement.